<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resouces/resource.qrc">
    <normaloff>:/logo/ico/pm_icon.ico</normaloff>:/logo/ico/pm_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#f8f8f8;
background-color:#181818;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Open Sans Semibold</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="createPasswordButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#383838;</string>
          </property>
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBoxShow">
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Show password</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_characterset">
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="text">
           <string>Character Set:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="passwordTemplateSwitch">
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#383838;</string>
          </property>
          <item>
           <property name="text">
            <string>All Characters</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alphabetical</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alphanumerical</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_length">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="text">
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_pwdLength">
          <property name="font">
           <font>
            <family>Open Sans Semibold</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#383838;</string>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="font">
         <font>
          <family>Open Sans Semibold</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#383838;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>createPasswordButton</tabstop>
  <tabstop>checkBoxShow</tabstop>
  <tabstop>passwordTemplateSwitch</tabstop>
  <tabstop>spinBox_pwdLength</tabstop>
 </tabstops>
 <resources>
  <include location="../../resouces/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
